prow/pubsub/subscriber/subscriber.go

```diff
@@ -231,7 +231,7 @@ func (prh *presubmitJobHandler) getProwJobSpec(cfg prowCfgClient, pc *config.InR
 		logger.Debug("Getting prow jobs.")
 		var presubmitsWithInrepoconfig []config.Presubmit
 		var err error
-		presubmitsWithInrepoconfig, err = pc.GetPresubmits(orgRepo, baseSHAGetter, headSHAGetters...)
+		presubmitsWithInrepoconfig, err = pc.GetPresubmits(orgRepo, refs.CloneURI, baseSHAGetter, headSHAGetters...)
```

Comment:
I think this is a breaking change since messages will now need to include the cloneURI to work with Gerrit, right?

Reply:
this is not really a breaking change, it's a requirement today to pass in cloneURI for inrepoconfig https://github.com/kubernetes/test-infra/blob/fe22f48cb484dbe81f466d7060f8fcfbe4de0f3a/prow/pubsub/subscriber/subscriber.go#L463